corral/util/error.pony

```diff
@@ -1,4 +0,0 @@
-class val Error
```

Comment:
Like in #133, this was only used to wrap error messages back to the callers, and its name clases with the log level.